gradle.properties

```diff
@@ -15,7 +15,7 @@
 #
 
 group=org.sonatype.gradle.plugins
-version=2.0.13-SNAPSHOT
+version=2.1.0-SNAPSHOT
```

Comment:
Question. Why has this changed?

Reply:
> Question. Why has this changed?

No particular reason beyond "I felt in the mood" 😅